controllers/storagecluster/cephobjectstores.go

```diff
@@ -123,6 +124,12 @@ func (obj *ocsCephObjectStores) ensureDeleted(r *StorageClusterReconciler, sc *o
 
 // createCephObjectStore creates CephObjectStore in the desired state
 func (r *StorageClusterReconciler) createCephObjectStores(cephObjectStores []*cephv1.CephObjectStore, instance *ocsv1.StorageCluster) error {
+	scaledObjects := &keda.ScaledObjectList{}
+	err := r.Client.List(r.ctx, scaledObjects)
```

Comment:
Is this a namespaced resource? I wonder if the list can be focused to our namespace.